src/index.js

```diff
@@ -54,20 +54,36 @@ addParseCloud();
 // "javascriptKey": optional key from Parse dashboard
 // "push": optional key from configure push
 
-function ParseServer(args) {
-  if (!args.appId || !args.masterKey) {
+function ParseServer({
+  appId,
+  masterKey,
+  databaseAdapter,
+  filesAdapter = new GridStoreAdapter(),
+  push,
+  loggerAdapter = new FileLoggerAdapter(),
+  databaseURI,
+  cloud,
+  collectionPrefix = '',
+  clientKey = '',
+  javascriptKey = '',
+  dotNetKey = '',
+  restAPIKey = '',
+  fileKey = 'invalid-file-key',
+  facebookAppIds = [],
+  enableAnonymousUsers = true,
+  oauth = {},
+  serverURL,
+}) {
```

Comment:
I like that syntax, learning ES6 at the same time you know... js/node is only my part-time pleasure.